UI/GameSettingsScreen.cpp

```diff
@@ -1563,6 +1575,7 @@ UI::EventReturn GameSettingsScreen::OnAudioDevice(UI::EventParams &e) {
 		g_Config.sAudioDevice.clear();
 	}
 	System_SendMessage("audio_resetDevice", "");
+	PostDarwinNotificationIfPossible("AudioConfigurationHasChanged", "CurrentDeviceWasChanged");
```

Comment:
If we need to keep this, it should be tied to `System_SendMessage()` only.  The implementation within `System_SendMessage()` would be allowed to call `PostDarwinNotificationIfPossible()`, but that would only be within SDL/ or ios/ code.

If there was any `PostMessageToWindowsMenubar()`, I would have an identical opinion.

That said, I agree with Henrik - if it's possible to update the menu items on some event prior to loading, that would be better.  Sounds like there is a way with NSMenuDelegate.

-[Unknown]